src/style.css

```diff
@@ -1,3 +1,89 @@
-body {
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+* {
+  padding: 0;
   margin: 0;
```

Comment:
- Don't use * selector (it impacts performance)
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/35af67d3-f18a-43d0-9ed8-48eda4921666)